apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -80,6 +88,14 @@ type TriggerAuthenticationSpec struct {
 	AzureKeyVault *AzureKeyVault `json:"azureKeyVault,omitempty"`
 }
 
+// TriggerAuthenticationStatus defines the observed state of TriggerAuthentication
+type TriggerAuthenticationStatus struct {
+	// +optional
+	ScaledObjectNamesStr string `json:"allscaledobjectnamesstring,omitempty"`
```

Comment:
Please rename `allscaledobjectnamesstring` to something a little bit shorter.  Maybe  just `scaledObjects` or `usedByScaledObjects`? Either way please use camelCase.

Reply:
Rename allscaledobjectnamesstring to scaledObjects